packages/devtools_app/test/scenes/performance/default.dart

```diff
@@ -0,0 +1,61 @@
+// Copyright 2022 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be found
+// in the LICENSE file.
+
+import 'dart:async';
+import 'dart:convert';
+import 'dart:io';
+
+import 'package:devtools_app/devtools_app.dart';
```

Comment:
extra devtools_app import?

Reply:
it is needed. generally I'd just trust our lints for unneeded imports to do their job and not comment on extra imports in a code review. I tested and in this case a bunch of symbols are unresolved if this import is removed.

Reply:
I think Dan was referring to the fact that we try to import files directly instead of importing devtools_app.dart, which exports a ton of files in DevTools

Reply:
I'd like to push back on this. in test code I don't see anything wrong with just importing devtools_app. maybe if you were lib/src/ui/common importing devtools_app would be ugly but it seems like the cleanest way to surface this from a test.